### DISCUSSION TITLE
Codespaces pricing for students

### DISCUSSION THREAD

**Discussion by OpenSourceSimon** [community/community#38759] [GitHub Education] [answered]
![image](https://user-images.githubusercontent.com/65854503/201054045-5729b385-1798-47a8-a667-8a0f769b9048.png)
Hey! I saw on the Codespaces page that it's free for students, but I can't find a specification for that. For how long is it free and with what specs?

**Comment by sarajcev** [comment, marked as answer]
Hi! It was announced just yesterday on the GitHub Universe Conference that the Codespaces are now available for individuals, including up to 60 hours free every month. This is a great news! I've tested the Codespaces today with my account (university pro) and it works, even directly from the VS Code. Several hardware specs were available, but I think this will depend on your region. I could use a machine with 16 CPU cores, 32 GB of RAM and 128 GB of disk space. More info on how to set it up can be found here: https://docs.github.com/en/codespaces/overview Hope this helps.

**Reply by OpenSourceSimon** [reply]
Thanks for your clear answer!

**Reply by devashish2024** [reply]
@sarajcev I have github pro with github student dev pack too, access to only 4 cores CPu, 16gb ram and 32gb storage. How can I get more better like you :P

**Comment by samus-aran** [comment]
Hi @OpenSourceSimon 👋 

Thanks @sarajcev for your informative answer! Super happy you're enjoying the usage of Codespaces 🎉  

Indeed the [latest announcement from GitHub Universe 2022](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/) is that free GitHub accounts receive 60 hours every month (using 2 cores). But I would like to add that the monthly amount of storage and core hours of usage available to students is [equivalent to the amount](https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/using-github-codespaces-with-github-classroom#about-github-codespaces) included with [GitHub Pro accounts](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#monthly-included-storage-and-core-hours-for-personal-accounts) which is 90 hours (using 2 cores) per month. 👍

**Reply by sarajcev** [reply]
That's more great news! Thanks for the info.

**Reply by Buedenbender** [reply]
Agreed on the great news but where does one apply to become a verified student?

**Reply by juanpflores** [reply]
You can apply here: https://education.github.com/
